templates/HAPROXY_HEAD

```diff
@@ -0,0 +1,18 @@
+global
+  daemon
+  log /dev/log local0
+  log /dev/log local1 notice
+  maxconn 4096
+  tune.ssl.default-dh-param 2048
+defaults
+  log               global
+  retries           3
+  maxconn           2000
+  timeout connect   5s
+  timeout client    50s
+  timeout server    50s
+listen stats
+  bind 0.0.0.0:9090
```

Comment:
I think in "the real world" a public slave wouldn't necessarily be accessible from the internet. You'd probably have an ELB in front of it. We actually already do this with AWS (we have a public slave ELB, although it's not properly configured).